### PR TITLE
Chinese annotation

### PR DESCRIPTION
**Pull request by VisualSJ**
@natural-law Please review this pr, Thanks

### CODE REVIEW

File: cocos/ui/UILayout.h [hunk 1]
```diff
@@ -47,47 +48,59 @@ class LayoutManager;
...
     
     /**
-     * @brief Get all elements of the layout.
-     *
-     * @return A vector of Node pointers.
+     * @brief @~english Get all elements of the layout.
+     * @~chinese 获取Layout内所有的元素。
+     * 
+     * @return @~english A vector of Node pointers.
+     * @~chinese 一个包含节点的指针的向量。
```
Comment (natural-law) [inline cocos/ui/UILayout.h]: 向量->数组

File: cocos/ui/UILayout.h [hunk 2]
```diff
@@ -147,215 +177,341 @@ class CC_GUI_DLL Layout : public Widget, public LayoutProtocol
...
+     * @return @~english if layout is clipping enabled.
+     * @~chinese 是否启用了布局是剪裁。
      */
     virtual bool isClippingEnabled()const;
     
     /**
+     * @~english
      * Returns the "class name" of widget.
+     * @~chinese 
+     * 返回“类名”的小部件。
```
Comment (natural-law) [inline cocos/ui/UILayout.h]: 应该是：组件的类名。

File: cocos/ui/UILayout.h [hunk 3]
```diff
@@ -379,27 +542,43 @@ class CC_GUI_DLL Layout : public Widget, public LayoutProtocol
     virtual void removeChild(Node* child, bool cleanup = true) override;
     
     /**
+     * @~english
      * Removes all children from the container with a cleanup.
      *
+     * @~chinese 
+     * 从容器中删除所有的孩子。
```
Comment (natural-law) [inline cocos/ui/UILayout.h]: 孩子->子节点

File: cocos/ui/UILayout.h [hunk 4]
```diff
@@ -493,105 +695,176 @@ class CC_GUI_DLL Layout : public Widget, public LayoutProtocol
...
+     * @~chinese 
+     * 当一个布局将焦点传递给它的子节点,则会使用这种方法来确定使用哪个算法传递（最近或最远的距离算法）
      */
     void findProperSearchingFunctor(FocusDirection dir, Widget* baseWidget);
     
     /**
+     * @~english
      * find the first non-layout widget in this layout
+     * @~chinese 
+     * 发现在这个layout中的第一个非布局部件
```
Comment (natural-law) [inline cocos/ui/UILayout.h]: 感觉应该翻译成：
查找这个布局中第一个不是 layout 类型的控件。

**Comment by natural-law** [comment]
@VisualSJ 
After all, I think widget should be translate to "控件" or "组件"， but not "部件". It's my opinion.

**Comment by VisualSJ** [comment]
@natural-law 
Thanks.
Known problems have been fixed. Please review this again~~

**Comment by natural-law** [comment]
@andyque 
It's ok for merge.

**Pull request merged**